code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1119,11 +1151,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/syndicatebomb/on_mob_life(mob/living/carbon/M)
 	if(is_syndicate(M))
+		to_chat(M, span_notice("The Syndicate will always Win!"))
```

Comment:
```suggestion
```
god no, this would cause it to spam the syndie agent

Reply:
Forgot to remove this part my bad